src/requestLooper.test.ts

```diff
@@ -90,4 +91,38 @@ describe('requestLooper', () => {
     scheduler.flush();
     expect(onCancel).toBeCalledTimes(1);
   });
+
+  it('increments the request id', (done) => {
```

Comment:
This test confirms that we use the passed in requestId and appends the count for successive async queries